maasjuju_toolkit/util.py

```diff
@@ -123,6 +123,9 @@ def query_machines(machine_filters):
     See examples in EXAMPLES.md"""
     rows = MaaSCache.select().order_by(MaaSCache.fqdn)
 
+    if isinstance(machine_filters, str):
```

Comment:
If you use `nargs='+'` and expect an input from the user like this `--destinations m1 m2 m3` then this will be unnecessary and you will feed query_machines with a list as machine_filters